packages/@lwc/features/src/__tests__/flags.spec.ts

```diff
@@ -163,7 +209,19 @@ pluginTester({
         featureFlags,
     },
     babelOptions,
-    tests: nonProdTests,
+    tests: {
+        ...nonProdTests,
+        'should throw an error if runtime flags are manually introduced': {
+            error:
+                'Runtime flags should never be used directly and should only be added by the compiler.',
```

Comment:
i feel like we should remove the second part of the message, i feel like it will add confusion. How about:
`Illegal syntax. Runtime flags cannot be used directly`

Reply:
Removed this assertion after some more thought...we really can't differentiate between the runtime flags added by the compiler vs by a human.